### PR TITLE
Add feed to project page

### PR DESCRIPTION
**Pull request by itexpert120**
resolves https://github.com/NEARBuilders/gateway/issues/353

https://github.com/NEARBuilders/gateway/assets/46527178/6aa3e0ee-be06-4a01-8289-609f39b5d79e

### CODE REVIEW

File: apps/new/widget/page/project/Index.jsx [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+  return "Loading...";
+}
+
 const { Layout } = VM.require(
   "${config_account}/widget/page.project.Layout",
 ) || {
   Layout: () => <></>,
 };
+
+const projectId = props.id.split("/")[2] ?? null;
```
Comment (Megha-Dev-19) [inline apps/new/widget/page/project/Index.jsx:19]: let's add it in `lib`, something like `getProjectIdFromProps`
I also used it in `tasks`, can you also update it there.
thanks

File: apps/new/widget/page/projects/Editor.jsx [hunk 2]
```diff
@@ -51,12 +51,16 @@ const isNearAddress = (address) => {
...
   { id: "overview", label: "Overview", disabled: true, checked: true },
-  { id: "activity", label: "Activity", checked: true },
+  // { id: "activity", label: "Activity", checked: true },
   { id: "tasks", label: "Tasks", checked: true },
   // Uncomment after the support is added
   // { id: "discussion", label: "Discussion", checked: false },
   // { id: "code", label: "Code", checked: false },
   // { id: "roadmap", label: "Roadmap", checked: false },
+  // Feed tabs
+  { id: "allFeed", label: "All Feed", checked: true },
```
Comment (Megha-Dev-19) [inline apps/new/widget/page/projects/Editor.jsx:61]: may be we can rename it to `activity`?
what do you think?

**Review by Megha-Dev-19 — APPROVED** [review]
LGTM, left few suggestion, and also sent text to Cecilia about design

**Comment by Megha-Dev-19** [comment]
image
@itexpert120 here instead of using the project ID in the heading, we can use the title of the project, since Id will always remain same but admin can change the title.

File: apps/new/widget/page/project/Index.jsx (hunk 1 above)
Comment (elliotBraem) [reply apps/new/widget/page/project/Index.jsx:19]: I like this too -- `getProjectIdFromPath`

File: apps/new/widget/page/projects/Editor.jsx (hunk 2 above)
Comment (elliotBraem) [reply apps/new/widget/page/projects/Editor.jsx:61]: ```{ id: "activity", label: "Activity", checked: true },```

**Comment by elliotBraem** [comment]
I agree with comments above, and resolve merge conflicts

**Pull request merged**